packages/react-charts/src/components/ChartLegendTooltip/ChartLegendTooltip.tsx

```diff
@@ -295,11 +295,8 @@ export interface ChartLegendTooltipProps extends ChartCursorTooltipProps {
   themeColor?: string;
   /**
    * The title prop specifies a title to render with the legend.
-   * This prop should be given as a string, or an array of strings for multi-line titles.
-   *
-   * Example: title={(datum) => datum.x}
    */
-  title?: string | string[] | Function;
+  title?: string;
```

Comment:
same question here about whether this would break people passing a one of the removed types.

Reply:
This must match the current Victory type in order to compile.

Reply:
Ahhh. So it is a breaking change from Victory then.

Reply:
Yes, not much we can do but match the Victory changes